wellcomecollection/wellcomecollection.org
# Make card line underline currentColor

## What does this change?
#10628 
Makes the card title underline color the same as the text (so that it should be as visible as the text is against its background).

__Before__
<img width="538" alt="before" src="https://github.com/user-attachments/assets/ea5ab9e5-089b-4d3e-99b6-ac2bc04e2c40">

__After__
<img width="526" alt="after" src="https://github.com/user-attachments/assets/7ff16698-85c6-4979-9528-fb77e81dfbb5">


## How to test
- Visit `/collections`
- Hover the cards on light/dark backgrounds and verify the underlines are the same colour as the text

## How can we measure success?
Users can see the underlines. Improves accessibility.

## Have we considered potential risks?
Can't think of any
